packages/astro-theme/components/LinkButton.astro

```diff
@@ -3,11 +3,10 @@ import { Icon } from "astro-icon";
 interface Props {
 	href: string;
 	label: string;
-	type?: "primary" | "primary2" | "secondary" | "discord";
+	type?: "primary" | "primary2" | "secondary" | "primary-cheerp" |"secondary-cheerp"| "discord";
```

Comment:
You could probably do away with this by changing the primary colour in the Tailwind config to #8e88ff for Cheerp. So all buttons and links change to the Cheerp theme, not just specific buttons

Reply:
I'm still unsure if I will use the color somewhere else that is not the home site, will have this in mind if I keep on using the color somewhere else as I progress with the design. Good idea.